modules/KIWIRuntimeChecker.pm

```diff
@@ -1470,6 +1477,43 @@ sub __checkDeprecatedFilesystem {
 }
 
 #==========================================
+# __checkMountDependencies
+#------------------------------------------
+sub __checkMountDependencies {
+    my $this = shift;
+    my $kiwi = $this->{kiwi};
+    my $msg;
+    if ($ENV{KIWI_IGNORE_OLD_MOUNTS}) {
+        return 1;
+    }
+    my $kiwi_mount_references = KIWIQX::qxx (
+        "grep -il kiwi /proc/[1-9]*/task/*/mounts|cut -d/ -f3|sort -u 2>&1"
```

Comment:
I wasn't sure if `kiwi` was the best thing to `grep` for here - but you know better than me :)